src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -423,4 +392,9 @@ def replace_jquery_meta_characters(input)
     # The stated characters are c&p from https://api.jquery.com/category/selectors/
     input.gsub(/[!"#$%&'()*+,.\/:\\;<=>?@\[\]^`{|}~]/, '_')
   end
+
+  def word_break(string, length)
+    # adds a <wbr> tag after an amount of given characters
+    string.scan(/.{1,#{length}}/).join("<wbr>").html_safe
```

Comment:
``
string.scan(/.{1,#{length}}/).join("<wbr>".html_safe)
``
And I would test that it does escape if you insert html!